Tests/test_file_png.py

```diff
@@ -103,6 +103,9 @@ def test_sanity(self):
         hopper("I").save(test_file)
         Image.open(test_file)
 
+        hopper("I;16").save(test_file)
+        Image.open(test_file)
```

Comment:
This is just a sanity test that it can save and re-open with no exceptions. 

Is it worth adding a new test method to actually check the file is saved and loaded correctly?

Reply:
@radarhere ^ What do you think?

Reply:
Okay, I've changed the tests, see what you think.

Under the hood, this is both I was being converted to I;16B. I am now also converting I;16 to I;16B. The downside is that when they are reloaded, they are both reloaded as I, since there is no difference.